src/license.ts

```diff
@@ -6,12 +6,21 @@ import chalk from "chalk";
 import Debug from "debug";
 import path from "path";
 import parse from "spdx-expression-parse";
-import satisfies from "spdx-satisfies";
+import spdxSatisfiesOriginal from "spdx-satisfies";
```

Comment:
@mvayngrib Rename to `spdxSatisfies`

Reply:
5d359a8442bd0f286ff7ee853962a42231653be5